.azure-pipelines/continuous-integration.yml

```diff
@@ -0,0 +1,53 @@
+name: $(date:yy)$(DayOfYear)$(rev:.r)
+
+trigger:
+  - master
+  - releases/shipped
+
+variables:
+  majorAndMinorVersion: '0.5'
+  revision: '$(Build.BuildNumber)'
+
+jobs:
+  - job: Build_Windows
+    variables:
+      platformFriendlyName: Windows
+    strategy:
+      maxParallel: 2
+      matrix:
+        #debug:
+        #  configuration: Debug

```

Comment:
Should we be building both Debug and Release?  Or should this be a config variable so we can chose for the build?  Having this commented out doesn't seem right if we have to uncomment and PR to run a Debug build.